### PR TITLE
Fix pipeline, emoji and syntax

### PR DESCRIPTION
**Pull request by anoadragon453**
Fixes https://github.com/matrix-org/dendrite/issues/697

Emoji syntax has `-` in some places and `_` in some places :')

`image` indentation was my bad whoopsie

### CODE REVIEW

**Comment by anoadragon453** [comment]
This'll require a review of golangci-lint's config. Also I took out `misspell` as it's included in `golangci-lint` but presumably we have to tell it which files/folders to run on.

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  fast: false
+
+
+issues:
+  # List of regexps of issue texts to exclude, empty list by default.
+  # But independently from this option we use default exclude patterns,
+  # it can be disabled by `exclude-use-default: false`. To list all
+  # excluded by default patterns execute `golangci-lint run --help`
+  exclude:
+    - abcdef
```
Comment (anoadragon453) [inline .golangci.yml]: Stuff like this should be commented out for instance.

**Comment by anoadragon453** [comment]
Note to self: consider pipeline order before merging. Should we be waiting on lint?

File: .golangci.yml [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+  disable:
+    - bodyclose
+    - depguard
+    - dupl
+    - gochecknoglobals
+    - gochecknoinits
+    - gocritic
+    - gofmt
+    - golint
+    - gosec
```
Comment (babolivier) [inline .golangci.yml]:
```suggestion
    - gosec # Should turn back on soon
```

**Pull request merged**